tests/test_plotting.py

```diff
@@ -297,6 +299,15 @@ def test_plot_surfaces_no_theta(DSHAPE_current):
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
+def test_plot_boundary():
+    """Test plotting boundary."""
+    eq = get("W7-X")
```

Comment:
@f0uriest @dpanici It might be worth changing all the plotting tests to use the `desc/examples/` files like this function does, instead of the `DSHAPE_current` solution we are presently using for everything. That would allow us to remove the `@pytest.mark.slow` and `@pytest.mark.solve` markers, and allow us to test on stellarator solutions.

Reply:
OK yeah it probably wouldn't save much time. But I do think it would be good to have more plotting tests use stellarator examples, because there is often different functionality for stellarators vs tokamaks.